lib/transform.js

```diff
@@ -44,6 +44,7 @@ module.exports = class Transform {
                     slotMap.set(attribs.slot, [node]);
                 }
                 this._pushText(node.next, slotMap.get(attribs.slot));
+                delete attribs.slot;
```

Comment:
I tested it in our code and object is always in fast mode even after calling delete on it.